cluster/gce/gci/configure-helper.sh

```diff
@@ -2445,6 +2471,10 @@ EOF
   if [[ "${ENABLE_DEFAULT_STORAGE_CLASS:-}" == "true" ]]; then
     setup-addon-manifests "addons" "storage-class/gce"
   fi
+  #if [[ "${FEATURE_GATES:-}" =~ "VolumeSnapshotDataSource=true" ]]; then
```

Comment:
we may want to have an ENABLE_SNAPSHOTS_CRDS flag (not feature gate based)

Reply:
Done